en/docs/develop/restapis/session.yaml

```diff
@@ -269,4 +387,4 @@ securityDefinitions:
     scopes:
       read: Grants read access
       write: Grants write access
-      admin: Grants read and write access to administrative information
+      admin: Grants read and write access to administrative information
```

Comment:
is this red icon after the word "information" intentional?

Reply:
That's due to missing the newline at the end of the file. I added it.